sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -678,6 +678,14 @@ public abstract partial class TokenCredential
         public string[] Scopes { get { throw null; } }
         public string? TenantId { get { throw null; } }
     }
+    public partial class UserAgentValue
+    {
+        public UserAgentValue(System.Type packageType, string? applicationId = null) { }
+        public string? ApplicationId { get { throw null; } }
+        public System.Type PackageType { get { throw null; } }
+        public void ApplyToMessage(Azure.Core.HttpMessage message) { }
```

Comment:
Do we need the `ToMessage` suffix?  Don't we usually call this just `Apply(HttpMessage)` in other APIs?

Reply:
It was initially just message, but the concern we discussed was that it could be unclear in which direction the application is being made. I think this is slightly more clear, but I don't feel super strongly